server/routes/index.js

```diff
@@ -0,0 +1,80 @@
+const busboy = require('connect-busboy');
+const helmet = require('helmet');
+const bodyParser = require('body-parser');
+const requestLanguage = require('express-request-language');
+const languages = require('../languages');
+const storage = require('../storage');
+const config = require('../config');
+const pages = require('./pages');
+// const lang = require('fluent-langneg')
+
+module.exports = function(app) {
+  app.use(
+    requestLanguage({
+      languages
+    })
+  );
+  app.use(helmet());
+  app.use(
+    helmet.hsts({
+      maxAge: 31536000,
+      force: config.env === 'production'
+    })
+  );
+  app.use(
+    helmet.contentSecurityPolicy({
+      directives: {
+        defaultSrc: ["'self'"],
+        connectSrc: [
+          "'self'",
+          'https://sentry.prod.mozaws.net',
+          'https://www.google-analytics.com'
+        ],
+        imgSrc: ["'self'", 'https://www.google-analytics.com'],
+        scriptSrc: ["'self'"],
+        styleSrc: ["'self'", 'https://code.cdn.mozilla.net'],
+        fontSrc: ["'self'", 'https://code.cdn.mozilla.net'],
+        formAction: ["'none'"],
+        frameAncestors: ["'none'"],
+        objectSrc: ["'none'"],
+        reportUri: '/__cspreport__'
+      }
+    })
+  );
+  app.use(
+    busboy({
+      limits: {
+        fileSize: config.max_file_size
+      }
+    })
+  );
+  app.use(bodyParser.json());
+  app.get('/', pages.index);
+  app.get('/legal', pages.legal);
+  app.get('/jsconfig.js', require('./jsconfig'));
+  app.get('/share/:id', pages.blank);
+  app.get('/download/:id', pages.download);
+  app.get('/completed', pages.blank);
+  app.get('/unsupported/:reason', pages.unsupported);
+  app.post('/api/upload', require('./upload'));
+  app.get('/api/download/:id', require('./download'));
+  app.get('/api/exists/:id', require('./exists'));
+  app.post('/api/delete/:id', require('./delete'));
```

Comment:
should this change have incremented the version number? (or if not, could it do so in the future?) I'm not presuming `/__version__` will always be available, but i was hoping to use it for the time being as a sanity check https://github.com/ehuggett/send-cli/issues/1

Reply:
The version number will change when we tag a release. For the master branch we might want to add a label to the version.json version to indicate that it may not be stable, for example "v1.1.2-master.53e822", meaning the release version will be some semver > 1.1.2. Would that help your situation?

Reply:
> The version number will change when we tag a release.

Its only just dawning on me how foolish attempting to support send.dev.mozaws.net by checking the version number is 🤣 Sorry! (I can't even claim i was half asleep)

It would make the check simpler, but I will check the value of commit AND version to determine this ie if (version == "v1.1.2" && commit == "0bf8481") {ok stable} else {warn about unstable} .
https://send.dev.mozaws.net/__version__ 
`{"commit":"ced640c","source":"https://github.com/mozilla/send/","version":"v1.1.2"}`

Which will also work if someone uses the --service-local or --service <url> for dev usage, and might still work on versions >1.1.2 if they are similar enough, so thanks for the suggestion, but I didn't realise i already had everything i needed to attempt "reliable" behaviour.